src/scss/custom.scss

```
@@ -4,29 +4,49 @@
  * work well for content-centric websites.
  */
 
+@import url('https://fonts.googleapis.com/css2?family=Manrope:wght@200..800&display=swap');
+
+@font-face {
+  font-family: 'PolySans';
```

Stylelint (17.11.1)

[error] Expected no quotes around "PolySans" (font-family-name-quotes)

(font-family-name-quotes)